MoviesTVSentiments/app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
```

Comment:
A small request, can we change the colors so they look more like our movies app?

We use "#D60B41" for colorPrimaryDark, and "#FF1744" for colorPrimary and colorPrimary.